include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -1174,6 +1174,21 @@ def RWProbeOp : FIRRTLExprOp<"ref.rwprobe",
   let assemblyFormat = "$target attr-dict `:` $type";
 }
 
+def RWProbeSSAOp : FIRRTLExprOp<"ref.rwprobe.ssa"> {
+  let summary = "FIRRTL RWProbe";
```

Comment:
A different short summary would be good given that this conflicts with the existing `RWProbeOp` summary. (I can't recall what this actually produces, though!)

Reply:
Oh, gosh, yeah totally! Thanks!